config/mtl.toml

```diff
@@ -597,3 +597,20 @@ count = 23
 	pin = [0, 0, 0xfeef, 0xf, 0xf, 0x45ff, 1, 0, 0xfeef, 0xf, 0xf, 0x1ff]
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 1000000, 128, 128, 0, 0, 0]
+
+	[[module.entry]]
+        name = "RTC_AEC"
+        uuid = "B780A0A6-269F-466F-B477-23DFA05AF758"
+        affinity_mask = "0x3"
+        instance_count = "1"
+        domain_types = "0"
+        load_type = "1"
+        module_type = "10"
+	init_config = "1"
+        auto_start = "0"
+        sched_caps = [1, 0x00008000]
+
+        # pin = [dir, type, sample rate, size, container, channel-cfg]
+        pin = [0, 0, 0x8, 0x2, 0x2, 0x1,
```

Comment:
Still no movement on breaking this apart into something readable?

Reply:
what is your expectation ? can you submit a feature in github ? @aiChaoSONG  is working on it. https://github.com/thesofproject/sof/pull/8243

Reply:
@cujomalainey this is a WIP, we are going to split and pre-process the tomls and pull in macros from headers and also use Kconfig. Plan is do have this done by end of Q4.